plugins/repository-hdfs/src/test/java/org/elasticsearch/repositories/hdfs/NoOpTests.java

```diff
@@ -0,0 +1,10 @@
+package org.elasticsearch.repositories.hdfs;
+
+import org.apache.lucene.util.LuceneTestCase;
+
+public class NoOpTests extends LuceneTestCase {
+
+    public void testDummy() {
+        // mmm, mmm, mmm, mmmhmm
```

Comment:
The `HdfsTests` is marked as third-party and thus doesn't gets executed when executing `gradle build`. Which then fails since there are not tests running - adding this dummy test prevents this from happening ('hey, here's a test').